src/pages/wishlist.js

```diff
@@ -197,7 +199,7 @@ const Wishlist = () => {
             <button
               id="addWishlist"
               onClick={addNewWishlist}
-              className={`w-2/3 bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded mt-7 ${
+              className={`w-2/3 bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded mt-7 disabled:bg-gray-600${
```

Comment:
Don't forget the space between `disabled:bg-gray-600` and `${`.
Same on line 214.